data/Makefile.am

```diff
@@ -33,8 +36,16 @@ desktop_DATA =						\
 	setup-shell.desktop				\
 	$(NULL)
 
+demovideodir = $(datadir)/eos-demo-mode
+demovideo_DATA = \
+	eos-demo-video.ogv \
+	eos-demo-video.desktop \
```

Comment:
Okay I've done the autostart check here, though I wasn't able to find much documentation on `AutostartCondition` by googling around and it seems like the two main operations here are `if-exists` and `unless-exists`. I've set this to `if-exists /home/demo-guest`, i.e, checking if the demo user's home directory exists, no matter which user we log in to.

Note that if a user somehow creates this username outside of the demo mode, the side effect is going to be that the demo video gets shown for all users. The only way we could prevent this is by installing a file in the home directory which is used by `AutostartCondition`, but then I feel like that is really no better than just installing the autostart launcher in the demo user's `~/.local/share/xdg/autostart` anyway.